cmd/kubeadm/app/phases/controlplane/manifests_linux_test.go

```diff
@@ -0,0 +1,217 @@
+// +build linux
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controlplane
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
```

Comment:
k8s uses go-cmd, but i don't think we have it under kubeadm currently?
maybe we do...

in any case, i think we should use "reflect"

```
if !reflect.DeepEqual(want, have) {
   t.Errorf("..., want: %+v", have: %+v, want, have)
}
```

Reply:
Done.